cmd/erasure-healing-common.go

```diff
@@ -270,6 +269,9 @@ func disksWithAllParts(ctx context.Context, onlineDisks []StorageAPI, partsMetad
 			if dataErrs[i] == nil {
 				// All parts verified, mark it as all data available.
 				availableDisks[i] = onlineDisk
+			} else {
+				// upon errors just make that disk's fileinfo invalid
+				partsMetadata[i] = FileInfo{}
```

Comment:
disksWithAllParts() is called only from one place, maybe we can set this outside disksWithAllParts() by checking the returned dataErrs, or make disksWithAllParts returns partsMetadata

Reply:
Correct @vadmeste that is the next refactor fr now this is okay